app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -94,6 +97,9 @@ def check_enrollments(enrollments)
 
       # Record the attempt to update the enrollment
       enrollment.update(status_check_attempted_at: status_check_attempted_at)
+
+      # Sleep for a while before we attempt to make another call to USPS
+      sleep request_delay_ms if idx < enrollments.length - 1
```

Comment:
The value of `request_delay_ms` is in seconds, so I think we should either rename the variable, or do the seconds conversion here.

```suggestion
      sleep request_delay_in_seconds if idx < enrollments.length - 1
```

Reply:
Done in [e4d180e](https://github.com/18F/identity-idp/pull/6959/commits/e4d180ecfcdc4992d21dca473f7c442dae60b3ef)